server/superdesk/publish/publicapi.py

```diff
@@ -47,7 +50,33 @@ def _transmit(self, queue_item, subscriber):
 
     def _fix_dates(self, item):
         for field, value in item.items():
-            if isinstance(value, dict) and '$date' in value:
-                item[field] = datetime.utcfromtimestamp(value['$date'] / 1000)
+            if isinstance(value, str):
+                try:
+                    new_value = datetime.strptime(value, config.DATE_FORMAT)
+                    item[field] = new_value
+                except:
```

Comment:
Should we only catch `ValueError` here? Or is it fine to silence an error if e.g. `DATE_FORMAT` does not exist in `config` for some reason?